docs/getting_started/tutorial.md

```diff
@@ -109,48 +122,52 @@ cd mysite
 The `requirements.txt` file contains all the dependencies needed in order to run the project.
 
 ### Create the database
 
-If you haven't updated the project settings, this will be a SQLite database file in the project directory.
+If you are yet to update the project settings, then your database is a SQLite database file in the project directory.
 
 ```sh
 python manage.py migrate
 ```
 
-This command ensures that the tables in your database are matched to the models in your project. Every time you alter your model (for example you may add a field to a model) you will need to run this command to update the database.
+This command ensures that the tables in your database matches the models in your project. Every time you alter your model, for example, if you add a field to a model, then you must run the `python manage.py migrate` command to update your database.
+
+```{note}
+If you wish to deploy your Wagtail site into production, then SQLite is not recommeded. This is because of its limited scalability and concurrency control, which makes it unsuitable for large-scale applications.
+```
 
 ### Create an admin user
 
 ```sh
 python manage.py createsuperuser
 ```
 
-This will prompt you to create a new superuser account with full permissions. Note the password text won’t be visible when typed, for security reasons.
+This will prompt you to create a new admin user account with full permissions. It's important to note that for security reasons, the password text won’t be visible while typing.
 
 ### Start the server
 
 ```sh
 python manage.py runserver
 ```
 
-If everything worked, <http://127.0.0.1:8000> will show you a welcome page:
+If everything works, <http://127.0.0.1:8000> shows you a welcome page:
 
 ![Browser screenshot of "Welcome to your new Wagtail site!" page, with teal egg above the title, and links to different resources. The page is shown inside a browswer tab, with browser URL bar at the top](../_static/images/tutorial/tutorial_1.png)
 
 ```{note}
-Throughout this tutorial we will use `http://127.0.0.1:8000` but depending on your setup, this could be `http://localhost:8000/` or a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct url for your local site.
+This tutorial uses `http://127.0.0.1:8000` as the URL for your development server but depending on your setup, this could be a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct url for your local site.
 ```
 
-You can now access the administrative area at <http://127.0.0.1:8000/admin>
+You can now access the [Admin Interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface) by logging into <http://127.0.0.1:8000/admin> with the username and password that you entered while creating an admin user through your command line terminal.
```

Comment:
Yes. It is very important. It helps with consistency and ensuring that the capitalization of our official names/concepts are in sync. For more information, read the [capitalization](https://developers.google.com/style/capitalization) section of the Google developer documentation style guide.

Reply:
It is also important that we provide links to relevant official names/concepts. This will help our new users quickly adjust to and understand the various names employed within the User guide documentation. It also helps maintain consistency with the names.

Reply:
We’ve discussed today and would prefer for this to avoid Title Case as this is a generic term (e.g. Django admin interface).